code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -693,6 +693,9 @@ func (c *Chain) addBlock(b *block.Block) *block.Block {
 		return eb
 	}
 	c.blocks[b.Hash] = b
+
+	c.pushBlockEvent(b)
```

Comment:
Oh, event db only exists in sharders, so we can not do this for miners.

Reply:
oooops